research/audioset/yamnet/export.py

```diff
@@ -106,7 +113,9 @@ def make_tf2_export(weights_path, export_dir):
 
   # Make TF2 SavedModel export.
   log('Making TF2 SavedModel export ...')
-  tf.saved_model.save(yamnet, export_dir)
+  tf.saved_model.save(
+    yamnet, export_dir,
```

Comment:
indent by 4 spaces for wrapped lines here and below and elsewhere.

Reply:
Yah, not sure why emacs on my laptop doesn't have solid python auto format.